xml/common_intro_lifecycle_support_i.xml

```diff
@@ -124,28 +124,28 @@
   <itemizedlist>
    <listitem>
     <para>
-     Technology previews
+     technology previews.
     </para>
    </listitem>
    <listitem>
     <para>
-     Sound, graphics, fonts, and artwork.
+     sound, graphics, fonts, and artwork.
     </para>
    </listitem>
    <listitem>
     <para>
-     Packages that require an additional customer contract.
+     packages that require an additional customer contract.
     </para>
    </listitem>
    <listitem>
     <para>
-     Some packages shipped as part of the module <emphasis>Workstation
+     some packages shipped as part of the module <emphasis>Workstation
      Extension</emphasis> are L2-supported only.
     </para>
    </listitem>
    <listitem>
     <para>
-     Packages with names ending in <package>-devel</package> (containing header
+     packages with names ending in <package>-devel</package> (containing header
```

Comment:
These make no sense to me. If there needs to be any punctuation at the end, I'd go for a comma, except on the last bullet. But I'd rather keep these as they were.